src/Store/SingleTableStore.php

```diff
@@ -95,7 +95,7 @@ public function has(string $aggregate, string $id): bool
         );
 
         if (!is_int($result) && !is_string($result)) {
-            throw new StoreException('invalid query return type');
+            throw new InvalidType('invalid query return type');
```

Comment:
This exception with the exact same message could be a dedicated one since it is used frequently

Reply:
how would you call them?